Doc/library/zipfile.rst

```diff
@@ -526,6 +526,9 @@ ZipFile Objects
       a closed ZipFile will raise a :exc:`ValueError`.  Previously,
       a :exc:`RuntimeError` was raised.
 
+   .. versionchanged:: 3.14
+      Now respects :envvar:`SOURCE_DATE_EPOCH` environment variable.
```

Comment:
I think the text can be improved here. The sentence lacks an object, but more importantly we don't say what has actually changed in the method's behaviour if SDE is set.

A

Reply:
```rst
.. versionchanged:: 3.14
  :meth:`.ZipFile.writestr` now respects the :envvar:`SOURCE_DATE_EPOCH` environment variable. If set, it uses this value as the modification timestamp for the file written into the ZIP archive.
```

This will better?

Reply:
Updated